packages/docusaurus-1.x/examples/basics/pages/en/users.js

```diff
@@ -34,10 +50,7 @@ class Users extends React.Component {
               <p>This project is used by many folks</p>
             </div>
             <div className="logos">{showcase}</div>
-            <p>Are you using this project?</p>
-            <a href={editUrl} className="button">
-              Add your company
-            </a>
+            <AddYourCompanySection siteConfig={siteConfig} />
```

Comment:
@jsjoeio Why not just add check as shown below?

```js
{siteConfig.repoUrl && (
  <>
    <p>Are you using this project?</p>
    <a
      href={`${siteConfig.repoUrl}/edit/master/website/siteConfig.js`}
      className="button">
      Add your company
    </a>
  </>
)}
```

Reply:
I was going with the original poster's idea and didn't think of this but good callout! I'll refactor and add a new commit.

Reply:
Yes, please do this, because creating a separate component is too "complicated" to fix such a minor bug.